scripts/gen_mmu.py

```diff
@@ -0,0 +1,538 @@
+import os
```

Comment:
Need to make sure you are using Python 3.
Please add: "#!/usr/bin/env python3"
as the first line of this file.